RELEASE-NOTES.txt

```diff
@@ -1,8 +1,14 @@
+1.11.0
+------
+* Toolbar scroll position now resets when its content changes.
+* Dark Mode for iOS.
+
 1.10.0
 ------
 * Adding a block from the post title now shows the add block here indicator.
 * Deselect post title any time a block is added
 * Fix loss of center alignment in image captions on Android
+* Fix merging of empty paragraph blocks when added from a post's title
```

Comment:
This shouldn't be here. It was originally included in 1.10.0, but @tug and I decided to revert it before that release. ([comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1246#discussion_r307836179), [commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1246/commits/d6a551ae140f26c49bb134144d6b38458723e68e)). Not sure why it's back, but I'll push a commit removing it.

Reply:
Thank you! 🙏